LootLockerServerSDK/Source/LootLockerServerSDK/Public/LootLockerServerForBlueprints.h

```diff
@@ -1573,14 +1583,4 @@ class LOOTLOCKERSERVERSDK_API ULootLockerServerForBlueprints : public UObject
     UFUNCTION(BlueprintPure, Category = "LootLockerServer Methods | Metadata", meta = (AdvancedDisplay = "Tags,Access", AutoCreateRefTerm = "Tags,Access"))
```

Comment:
Just writing one comment here, but it is in multiple places:
All `@param Access ` needs to be updated to mention something about `player.read`

Reply:
Fixed